ui/page/root/home_page.go

```diff
@@ -102,6 +102,11 @@ func (hp *HomePage) OnNavigatedTo() {
 	hp.ctx, hp.ctxCancel = context.WithCancel(context.TODO())
 
 	if hp.CurrentPage() == nil {
+		// start wallet sync
+		pg := NewWalletSelectorPage(hp.Load)
+		pg.OnNavigatedTo()
+		pg.OnNavigatedFrom()
```

Comment:
This is really hacky lol.

The only thought coming to mind is to load the wallets, and have them sync in background pending when we view the loaded wallets on the wallets page.